packages/scandipwa/src/component/FieldForm/FieldForm.component.js

```diff
@@ -19,7 +19,7 @@ import Form from 'Component/Form';
 import './FieldForm.style';
 
 /** @namespace Component/FieldForm/Component */
-export class FieldForm extends PureComponent {
+export class FieldForm extends Component {
```

Comment:
What is the reason to extend from `Component` instead of `PureComponent`?

Reply:
Because in MyAccountNewsletterSubscription.component used shouldComponentUpdate, for this function use of Component and not PureComponent and since it extends from FieldForm I replaced PureComponent in FieldForm
Return back, or leave as is?

Reply:
But now `FieldForm` isn't pure and you didn't provide the default `shouldComponentUpdate` for it.
I see two solutions:
* Provide a default `shouldComponentUpdate` in the base class that acts like `PureComponent`
* Refactor `MyAccountNewsletterSubscription` component, so it doesn't extend but reuse `FieldForm` component. Then `MyAccountNewsletterSubscription` can extend `Component`, while `FieldForm` will remain `PureComponent`.

Reply:
1. I'm not sure how to do this, in any case if we add shouldComponentUpdate, we need to change the extension to Component, which means the FieldForm is not pure, no?
2. Tried to separate, too much code needs to be duplicated

3. Would it be ok to just move shouldComponentUpdate to the container? Thus, only MyAccountNewsletterSubscriptionContainer will not be clean, and this container is not inherited from anyone, and will not affect anyone

Reply:
> Would it be ok to just move shouldComponentUpdate to the container?
Sure. If a container doesn't rerender a lot, the component doesn't rerender too.